dao/src/main/java/uk/gov/moj/sdt/dao/IndividualRequestDao.java

```diff
@@ -96,21 +96,22 @@ public IndividualRequestDao(final IndividualRequestRepository crudRepository, En
     public IIndividualRequest getIndividualRequest(final IBulkCustomer bulkCustomer, final String customerReference,
                                                    final int dataRetention) throws DataAccessException {
         if (LOGGER.isDebugEnabled()) {
-            LOGGER.debug("Get individual request matching the bulk customer[" + bulkCustomer + "], " +
-                    "customer reference[" + customerReference + "] and the data retention period[" + dataRetention +
-                    "]");
+            LOGGER.debug("Get individual request matching the bulk customer[{}], customer reference[{}] "
+                             + "and data retention period[{}]",
+                         bulkCustomer, customerReference, dataRetention);
         }
 
-        Predicate sdtCustomerPredicate = criteriaBuilder.equal(root.get("bulkSubmission").get("bulkCustomer").get("sdtCustomerId"),
+        Predicate sdtCustomerPredicate =
+            criteriaBuilder.equal(root.get("bulkSubmission").get("bulkCustomer").get("sdtCustomerId"),
                                                                bulkCustomer.getSdtCustomerId());
-        Predicate customerReferencePredicate = criteriaBuilder.equal(criteriaBuilder.lower(root.get("customerRequestReference")),
+        Predicate customerReferencePredicate =
+            criteriaBuilder.equal(criteriaBuilder.lower(root.get("customerRequestReference")),
                                                                      customerReference.toLowerCase());
```

Comment:
This is the second parameter of `equals()`, so should be in line with the first (`criteriaBuilder.lower(root.get("customerRequestReference"))`).